templates/project/bin/console.twig

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Sonata\{{ bundle_name }}\Tests\App\AppKernel;
```

Comment:
I need to be able to generate this bundle_name, do you know if we already have this information somehow in dev-kit? @VincentLanglet